pkg/config/config.go

```diff
@@ -823,6 +823,8 @@ func InitConfig(config Config) {
 	// Datadog security agent (runtime)
 	config.BindEnvAndSetDefault("runtime_security_config.enabled", false)
 	config.SetKnown("runtime_security_config.fim_enabled")
+	config.BindEnvAndSetDefault("runtime_security_config.erpc_dentry_resolution_enabled", true)
```

Comment:
What if one disable both ?

Reply:
Added some logic to make sure that the map resolution is set to `true` if both options are set to `false` 👍🏻